tools/reviewer/web/src/assets/code-review.proto

```diff
@@ -135,14 +139,15 @@ message Diff {
   // The main author of the diff, though certain snapshots can be made by others.
   Author author = 2;
   repeated Reviewer reviewer = 3;
-  string description = 4;
-  string bug = 5;
-  Status status = 6;
-
-  string workspace = 7;
-  int64 created_timestamp = 8;
-  int64 modified_timestamp = 9;
-  repeated Thread thread = 10;
+  repeated string cc = 4; // email list
```

Comment:
You don't have to change the code, but just know that any binary proto that you try to load, assumes that you're loading it with the same IDs with which it was created.
So for example, `string bug = 5;` changed to `string description = 5;` so if you try to load old binary data, field #5 with bug data will be put into the description.
In many cases though, you won't be able to read the data at all, since the types changed - i.e try to load a string into and int.
Right now it's alright since we can delete all data in the db anyway. Once we start to have meaningful data, we can't change it anymore.

Reply:
Ok, I'll be aware of it.
Actually, it's a relief. It was really annoying to change the indexes all the time.